include/sycl/algorithm/find.hpp

```diff
@@ -185,15 +187,17 @@ InputIt find_impl(ExecutionPolicy &snp, InputIt b, InputIt e,
   auto map = [=](size_t pos, value_type x) {
     return (p(x)) ? pos : size;
   };
-  auto red = [=](size_t x, size_t y){return std::min(x, y);};
+
+  auto red = [=](size_t x, size_t y){
```

Comment:
Lacking space between `){`.
By the way, is it required to use `red` instead of just passing `std::min` in the next line?